CONTRIBUTING.md

```diff
@@ -96,11 +95,9 @@ The subject contains succinct description of the change:
 * no dot (.) at the end
 
 ### Body
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+Wrap the body at 72 characters whenever possible (for example, don't modify backtraces to follow this rule). Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
 The body should include the motivation for the change and contrast this with previous behavior.
 
-### Footer
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
+The body contains (in order of appearance) a detailed **description** of the committed changes, references GitHub issues that the commit **closes** and any **breaking changes**. The **description** is mandatory, **closes** and **breaking changes** are optional.
 
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+**Breaking changes** should start with the phrase `BREAKING CHANGE:` followed by a line break. The rest of the commit message is then used for this.
```

Comment:
It doesn't break it, the parser just needs to be edited. See here for example

https://github.com/angular/angular.js/blob/c5cba6e9c15aa2d6cd065b4fba7166cc58d37c84/changelog.js#L44

I'd say that optional formats just complicate the instructions. Just `BREAKING CHANGE:` + line break sounds more reasonable.